lib/blueprinter/view_collection.rb

```diff
@@ -49,6 +49,11 @@ def sortable_fields(view_name)
         fields.delete(name)
       end
 
+      views[view_name].excluded_view_names.each do |excluded_view_name|
```

Comment:
I don't believe this is properly excluding all fields from a specified `exclude_view`.

May be worth adding this test into `base_render_examples`:

```ruby
  context 'Given blueprint has excluded ::views' do
    let(:exclude_minimal) do
      ['{"id":' + obj_id + '', '"description":"A person"',
      '"employer":"Procore"', '"position":"Manager"}'].join(',')
    end
    let(:exclude_normal) do
      ['{"id":' + obj_id + '', '"description":"A person"}'].join(',')
    end
    let(:blueprint) do
      Class.new(Blueprinter::Base) do
        identifier :id
        view :minimal do
          fields :first_name, :last_name
        end
        view :normal do
          include_view :minimal
          fields :position
          field :company, name: :employer
        end
        view :extended do
          include_view :normal
          field :description
        end
        view :exclude_minimal do
          include_view :extended
          exclude_view :minimal
        end

        view :exclude_normal do
          include_view :extended
          exclude_view :normal
        end
      end
    end
    it('returns json with view fields excluded') do
      expect(blueprint.render(obj, view: :exclude_minimal)).to eq(exclude_minimal)
      expect(blueprint.render(obj, view: :exclude_normal)).to eq(exclude_normal)
    end
  end
```
You'll notice this test will fail for this code as `exclude_view :normal` won't actually exclude all the fields in `normal`.

Reply:
@mcclayton , when default views come into picture, I believe it expected to be always included in the final render regardless of any exclude_views (containing the default view). can you share your thoughts on this.

Reply:
IMO default view should always be included as you suggested @Abhinaya-P unless `exclude_view :default` is specifically specified.

Reply:
Ahh yes I agree that default view should always be included (also unless `exclude_view :default`) is specifically provided 👍

Reply:
@mcclayton - I have included the functionality to handle default views and added few more tests. A thorough review and testing would be helpful here. Thanks.